python/lsst/ap/pipe/ap_pipe.py

```diff
@@ -196,373 +99,66 @@ def parsePipelineArgs():
     # Parse command line arguments with argparse
     parser = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter,
                                      description=textwrap.dedent('''
-    Process raw decam images with MasterCals from ingestion --> difference imaging
+    Process raw decam images with MasterCals from basic reduction --> difference imaging
```

Comment:
Keep this consistent with what the very first docstring says: "from basic processing --> source association"